Compile error in cactus_rt: signal_handler.cc

Hello, first of all thank you for this very helpful code, which I am trying to adapt for a real-time controls application. 

I downloaded cactus_rt and am trying to compile it on Linux Debian 12. When 'make debug' is invoked I receive the following error:

          signal_handler.cc:34:10: error: 'ignore' is not a member of 'std'

Commenting out line 34 allows compilation, but I am confused why the compiler complains about 'ignore'?

Thanks!

-Raul

--- a/src/cactus_rt/signal_handler.cc
+++ b/src/cactus_rt/signal_handler.cc
@@ -6,6 +6,7 @@
 #include <cstring>
 #include <stdexcept>
 #include <string>
+#include <tuple>
 
 namespace cactus_rt {
 /// @private

